### ISSUE TITLE
Iris is incompatible with Carpet.

### ISSUE THREAD

**Issue by Fleepe** [IrisShaders/Iris#2367]
### Minecraft Version

Minecraft 1.20.1

### Iris Version

iris-1.7.1+mc1.20.1.jar

### Sodium Version

sodium-fabric-0.5.10+mc1.20.1.jar

### Operating System

Windows 11

### What is your GPU?

Nvidia GeForce RTX 4060 Laptop

### Java Version

Java 21

### Reproduction Steps

1. Download Sodium, Iris and Carpet;
2. Try to launch Minecraft 1.20.1;
3. Get game crashed.

### Crash Report file and latest.log

```shell
---- Minecraft Crash Report ----
// I bet Cylons wouldn't have this problem.

Time: 2024-06-18 12:32:06
Description: Initializing game

java.lang.RuntimeException: Mixin transformation of net.minecraft.class_761 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at net.minecraft.class_310.<init>(class_310.java:583)
	at net.minecraft.client.main.Main.main(Main.java:211)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
	... 12 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [carpet.mixins.json:LevelRenderer_pausedShakeMixin from mod carpet] from phase [DEFAULT] in config [carpet.mixins.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379)
	... 15 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Variable modifier target for net/minecraft/class_761::changeTickPhase was removed by another injector [INJECT Applicator Phase -> carpet.mixins.json:LevelRenderer_pausedShakeMixin from mod carpet -> Apply Injections ->  -> Inject -> carpet.mixins.json:LevelRenderer_pausedShakeMixin from mod carpet->@ModifyVariable::changeTickPhase(F)F]
	at org.spongepowered.asm.mixin.injection.modify.ModifyVariableInjector.inject(ModifyVariableInjector.java:169)
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:277)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:445)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1377)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1062)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:402)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:327)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:422)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 15 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at net.minecraft.class_310.<init>(class_310.java:583)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Расширенная библиотека API Windows 32:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:Библиотека элементов управления взаимодействия с пользователем:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:API32 криптографии:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		CoreMessaging.dll:Microsoft CoreMessaging Dll:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.22621.3593 (WinBuild.160101.0800):Microsoft Corporation
		DEVOBJ.dll:Device Information Set DLL:10.0.22621.2506 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:Динамическая библиотека API DNS-клиента:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		GLU32.dll:Библиотека подпрограмм OpenGL:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.22621.3374 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:API вспомогательного приложения IP:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Библиотека клиента Windows NT BASE API:10.0.22621.3374 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Библиотека клиента Windows NT BASE API:10.0.22621.3374 (WinBuild.160101.0800):Microsoft Corporation
		MMDevApi.dll:MMDevice API:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		MSCTF.dll:Серверная библиотека MSCTF:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.24050.7 (b60bcb9c79f050335eb91db16f7d3e97336ad10e-dirty):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		OLEAUT32.dll:OLEAUT32.DLL:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE для Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		OpenAL.dll:Main implementation library:1.21.1:
		POWRPROF.dll:DLL модуля поддержки профиля управления питанием:10.0.22621.3374 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Модуль поддержки данных производительности Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Библиотека удаленного вызова процедур:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		SETUPAPI.dll:Windows Setup API:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Общая библиотека оболочки Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Многопользовательская библиотека клиента USER API Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.36.32532.0:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Службы HTTP Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		WINSTA.dll:Winstation Library:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		WINTRUST.dll:Microsoft Trust Verification APIs:10.0.22621.3737 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:32-разрядная библиотека Windows Socket 2.0:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		WTSAPI32.dll:Windows Remote Desktop Session Host Server SDK APIs:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.22621.3527 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Библиотека криптографических примитивов Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		cfgmgr32.dll:Configuration Manager DLL:10.0.22621.2506 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM для Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		cryptnet.dll:Crypto Network Related API:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		dinput8.dll:Microsoft DirectInput:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		drvstore.dll:Driver Store API:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		dwmapi.dll:Интерфейс API диспетчера окон рабочего стола (Майкрософт):10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		dxcore.dll:DXCore:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		extnet.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		fwpuclnt.dll:API пользовательского режима FWP/IPsec:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		icm32.dll:Microsoft Color Management Module (CMM):10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		imagehlp.dll:Windows NT Image Helper:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		inputhost.dll:InputHost:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		javaw.exe:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		jli.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		jna3133675534552384153.dll:JNA native library:6.1.4:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		jvm.dll:OpenJDK 64-Bit server VM:21.0.3.0:Eclipse Adoptium
		kernel.appcore.dll:AppModel API Host:10.0.22621.2715 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		lwjgl_opengl.dll
		lwjgl_stb.dll
		management.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		management_ext.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		msasn1.dll:ASN.1 Runtime APIs:10.0.22621.2506 (WinBuild.160101.0800):Microsoft Corporation
		mscms.dll:DLL-библиотека системы сопоставления цветов Майкрософт:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		msvcp140.dll:Microsoft® C Runtime Library:14.36.32532.0:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.22621.3374 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.22621.2506 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Расширение поставщика службы API Microsoft Windows Sockets 2.0:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:Поставщик оболочки совместимости для имен электронной почты:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Маршрутизатор Windows NCrypt:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		nio.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:Системная библиотека NT:10.0.22621.3374 (WinBuild.160101.0800):Microsoft Corporation
		ntmarta.dll:Поставщик Windows NT MARTA:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		nvgpucomp64.dll:NVIDIA GPU Compiler Driver, Version 555.99 :32.0.15.5599:NVIDIA Corporation
		nvoglv64.dll:NVIDIA Compatible OpenGL ICD:32.0.15.5599:NVIDIA Corporation
		nvspcap64.dll:NVIDIA Game Proxy:10,0,1,253:NVIDIA Corporation
		opengl32.dll:OpenGL Client DLL:10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Библиотека объектов производительности системы Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:Поставщик пространства имен PNRP:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.22621.3527 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Библиотека небольших программ оболочки:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		textinputframework.dll:"TextInputFramework.DYNLINK":10.0.22621.3672 (WinBuild.160101.0800):Microsoft Corporation
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.22621.3593 (WinBuild.160101.0800):Microsoft Corporation
		uxtheme.dll:Библиотека тем UxTheme (Microsoft):10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.36.32532.0:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
		win32u.dll:Win32u:10.0.22621.3733 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:API хранения Microsoft WinRT:10.0.22621.3527 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.22621.1 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Библиотека DLL основных типов Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		wldp.dll:Политика блокировки Windows:10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.22621.3374 (WinBuild.160101.0800):Microsoft Corporation
		xinput1_4.dll:API общего контроллера (Майкрософт):10.0.22621.2860 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:21.0.3.0:Eclipse Adoptium
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:211)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)

-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.3, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 1285554176 bytes (1226 MiB) / 2183135232 bytes (2082 MiB) up to 6442450944 bytes (6144 MiB)
	CPUs: 20
	Processor Vendor: GenuineIntel
	Processor Name: 12th Gen Intel(R) Core(TM) i7-12700H
	Identifier: Intel64 Family 6 Model 154 Stepping 3
	Microarchitecture: Alder Lake
	Frequency (GHz): 2.69
	Number of physical packages: 1
	Number of physical CPUs: 14
	Number of logical CPUs: 20
	Graphics card #0 name: NVIDIA GeForce RTX 4060 Laptop GPU
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x28e0
	Graphics card #0 versionInfo: DriverVersion=32.0.15.5599
	Graphics card #1 name: Intel(R) UHD Graphics
	Graphics card #1 vendor: Intel Corporation (0x8086)
	Graphics card #1 VRAM (MB): 1024.00
	Graphics card #1 deviceId: 0x4626
	Graphics card #1 versionInfo: DriverVersion=31.0.101.4502
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 26251.12
	Virtual memory used (MB): 10471.41
	Swap memory total (MB): 10240.00
	Swap memory used (MB): 159.15
	JVM Flags: 19 total; -XX:+UnlockExperimentalVMOptions -XX:+UnlockDiagnosticVMOptions -XX:+UseZGC -XX:+ZGenerational -XX:AllocatePrefetchStyle=1 -XX:MaxTenuringThreshold=1 -XX:NmethodSweepActivity=1 -XX:+AlwaysActAsServerClassMachine -XX:+AlwaysPreTouch -XX:+DisableExplicitGC -XX:-DontCompileHugeMethods -XX:+PerfDisableSharedMem -XX:+UseFastUnorderedTimeStamps -XX:+UseCriticalJavaThreadPriority -XX:+EagerJVMCI -XX:+PerfDisableSharedMem -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx6144m
	Loaded Shaderpack: (off)
	Launched Version: 1.20.1
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce RTX 4060 Laptop GPU/PCIe/SSE2 GL version 3.2.0 NVIDIA 555.99, NVIDIA Corporation
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: 20x 12th Gen Intel(R) Core(TM) i7-12700H
```

### Additional context

[latest.log](https://github.com/user-attachments/files/15883740/latest.log)
[crash-2024-06-18_12.32.06-client.txt](https://github.com/user-attachments/files/15883745/crash-2024-06-18_12.32.06-client.txt)

*If this is Carpet's fault, then I have no choice but to turn off Iris.*

**Comment by Dalethium** [comment]
I can confirm this issue is present, and it affects more than just carpet, along with the fact that this was not present in the previous version of Iris.

**Comment by esuity** [comment]
The bug is recurrent.Hope it can be solved.

**Comment by Poopooracoocoo** [comment]
As a workaround, you can downgrade Iris to 1.7.0, Sodium to 0.5.8, and Indium to 1.0.31.

**Comment by peow** [comment]
can confirm

**Comment by Flash-Git** [comment]
Would love a fix on this side of things, seems way more likely than hoping carpet updates 1.20.1.

**Comment by Viper4K** [comment]
i will bump it, since it has a big impact not just for me 
if this got acknowledged by iris devs, please leave a comment, because this bug nukes possibility of playing distant horizons with tons of mods

**Comment by Poopooracoocoo** [comment]
https://github.com/IrisShaders/Iris/commit/a4cc37ce7bc960b7aa28e5ea54fccb4ddbfed347 
The next release should fix it

**Comment by Fleepe** [comment]
Good

**Comment by Viper4K** [comment]
love to hear that, tysm!

**Comment by Fleepe** [comment]
I'm closing it since the problem has been fixed in iris 1.7.2
